pkg/networkmanager/interfaces.js

```diff
@@ -1620,7 +1632,7 @@ export function is_interface_connection(iface, connection) {
 }
 
 export function is_interesting_interface(iface) {
-    return !iface.Device || is_managed(iface.Device);
+    return !iface.Device || (is_managed(iface.Device) && iface.Device.DeviceType != "loopback");
```

Comment:
So you explicitly want `lo` to be "interesting"? I'm not sure how/where is_interesting_interface() is called and how it relates to skipping "lo" in the NetworkPage render function below, just wanted to make sure that this is really intended.

Reply:
> So you explicitly want `lo` to be "interesting"?

No, it should not be "interesting". `iface.Device.DeviceType != "loopback"`.

Reply:
`is_interesting` is used for the "Add bond" dialog, for example.  The user will only be offered "interesting" interfaces as potential bond members.

Reply:
So shouldn't you add the "lo" name special case for older OSes here as well?

Reply:
On older OSes, "lo" is not managed so that is enough to make it uninteresting.  Yeah, this is a bit tricky...